### PR TITLE
Adapting the tutorial island guide to focus on Redstone

### PR DESCRIPTION
**Pull request by ldunnplaymint**

### CODE REVIEW

File: tutorial/REFERENCE.md [hunk 1]
```diff
@@ -77,14 +77,14 @@ _options explained_
 #### Command line signing
 
 >[!CAUTION]
->Do not use a private key for any account you care about. You can use the Downstream connection dialogue box to copy the private key if connected with a burner. Or you can leave this option out and ds will give you a wallet connect QR code to sign.
+>It is advised to use a Private Key against Redstone due to security concerns. If you are working in a loval dev environment, make sure you are not using a Private Key for an account you care about or use the burner account system to generate a throwaway private key.
```
Comment (5p0rt5BEArD) [inline tutorial/REFERENCE.md:80]: "It is strongly advised NOT to use a Private Key against Redstone due to security concerns. This option is intended for use with local dev environment only. If you are working..."

**Review by 5p0rt5BEArD — DISMISSED** [review]
Good stuff
Only a couple of change requests
A future task I think is to combine the redstoneguide with tutorial one because they overlap.

File: contracts/src/maps/tutorial-room-2/README.md [hunk 2]
```diff
@@ -13,21 +13,14 @@ Once complete, your map should something look like this:
...
-Follow steps 1 through 4 of `tutorial-room-1` to get started. This will walk you through:
-- Deploying the game
-- Spawning a unit
-- Creating a map
-- Deploying new tiles
 
-[tutorial-room-1/README.md](../tutorial-room-1/README.md)
-
-The only change we want to make to these steps is in _Creating a map_. As this is the next room in the tutorial series, we will be creating our map at a different angle so it doesn't overlap with tutorial-room-1.
+We are going to put together the map below, it is the second spoke of tutorial Island! There is intentionally a choke point so we can show off Doors!
```
[the remark this answers is not in the document]
Comment (ldunnplaymint) [reply contracts/src/maps/tutorial-room-2/README.md]: Done

**Comment by ldunnplaymint** [comment]
> Good stuff Only a couple of change requests A future task I think is to combine the redstoneguide with tutorial one because they overlap.

Yeah, you are right. I wrote it like this because the milestone goal only goes as far as deploying tiles. I wanted to then get ahead of myself and re-write everything else because I think we just will end up doing so anyway!

**Review by ldunnplaymint — COMMENTED** [review]
addressed

**Pull request merged**